MAINTAINERS.md

```
@@ -15,4 +15,4 @@ This document lists the maintainers of the **SupportUsButton** project.
 * **Security Vulnerabilities**: For private security reports, please email [reach2saksham2004@gmail.com](mailto:reach2saksham2004@gmail.com) directly before public disclosure.
 * **Architectural Questions & Governance**:
   1. **AOSSIE Discord Channel**: [#support-us-button](https://discord.gg/hjUhu33uAn)
-  2. **GitHub Issues**: [github.com/AOSSIE-Org/SupportUsButton/issues](https://github.com/AOSSIE-Org/SupportUsButton/issues)
+  2. **GitHub Issues**: [github.com/AOSSIE-Org/SupportUsButton/issues](https://github.com/AOSSIE-Org/SupportUsButton/issues)
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)